src/uucore/Cargo.toml

```diff
@@ -24,7 +24,7 @@ thiserror = { version="1.0", optional=true }
 lazy_static = { version="1.3", optional=true }
 nix = { version="<= 0.13", optional=true }
 platform-info = { version="<= 0.1", optional=true }
-time = { version="<= 0.1.42", optional=true }
+time = { version="<= 0.1.43", optional=true }
```

Comment:
this is a necessary update?

Reply:
@sylvestre I otherwise had issues with using the new `chrono` releases. I mentioned this previously in my PR body text.